Formula/nasm.rb

```diff
@@ -1,8 +1,8 @@
 class Nasm < Formula
   desc "Netwide Assembler (NASM) is an 80x86 assembler"
   homepage "https://www.nasm.us/"
-  url "https://www.nasm.us/pub/nasm/releasebuilds/2.15.01/nasm-2.15.01.tar.xz"
-  sha256 "28a50f80d2f4023e444b113e9ddc57fcec2b2f295a07ce158cf3f18740375831"
+  url "https://www.nasm.us/pub/nasm/releasebuilds/2.15.02/nasm-2.15.02-xdoc.tar.xz"
+  sha256 "a2ec81aeb86b61dee8e43bdc561449119a65fb429efa48abdc569dc7471b1981"
```

Comment:
`https://www.nasm.us/pub/nasm/releasebuilds/2.15.02/nasm-2.15.02-xdoc.tar.xz` — it's documentation, the source itself:

```suggestion
  url "https://www.nasm.us/pub/nasm/releasebuilds/2.15.02/nasm-2.15.02.tar.xz"
  sha256 "f4fd1329b1713e1ccd34b2fc121c4bcd278c9f91cc4cb205ae8fcd2e4728dd14"
```

Reply:
Oh, thanks for spotting this 🤦